src/awsexplorer/regionNode.ts

```diff
@@ -30,19 +28,27 @@ export class RegionNode extends AWSTreeNodeBase {
         return this.info.regionName
     }
 
-    public constructor(info: RegionInfo) {
+    public constructor(info: RegionInfo, regionProvider: RegionProvider) {
         super(info.regionName, TreeItemCollapsibleState.Expanded)
         this.contextValue = 'awsRegionNode'
         this.info = info
         this.update(info)
 
-        this.cloudFormationNode = new CloudFormationNode(this.regionCode)
-        this.lambdaNode = new LambdaNode(this.regionCode)
-        this.schemasNode = new SchemasNode(this.regionCode)
+        if (regionProvider.isServiceInRegion('cloudformation', info.regionCode)) {
+            this.childNodes.push(new CloudFormationNode(this.regionCode))
+        }
+
+        if (regionProvider.isServiceInRegion('lambda', info.regionCode)) {
```

Comment:
Can we do this is a less copy paste way? maybe like have a list of services we support?